go.sum

```diff
@@ -30,6 +30,10 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible h1:dvc1KSkIYTVjZgHf/CTC2diTYC8PzhaA5sFISRfNVrE=
 github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible/go.mod h1:J2gT2udsDAN96Uj4KfcMRqY0/ypR+oyYUYmja8H+y+w=
+github.com/docker/docker v1.13.1 h1:IkZjBSIc8hBjLpqeAbeE5mca5mNgeatLHBy3GO78BWo=
+github.com/docker/docker v1.13.1/go.mod h1:eEKB0N0r5NX/I1kEveEz05bcu8tLC/8azJZsviup8Sk=
```

Comment:
This seems like a super old version of the docker client library?  Should it be moby/moby ?

Reply:
The docker deps are super weird. First of all they seemed to stop tagging at 17.03. Then when trying to use 17.03 it isn't picked up as a tag. As far as I could tell, people tend to use v1.13.1 often. I really don't know what we should do here.

Reply:
I definitely think we should follow up on this and figure out how docker expects us to use the latest packages. `moby/moby` straight up didn't compile for me.